src/main/scala/info/bethard/timenorm/TimeNormScorer.scala

```diff
@@ -15,6 +15,14 @@ object TimeNormScorer {
   case class Timex(id: String, textSpan: (Int, Int), time: TimeExpression) {
     val (textStart, textEnd) = textSpan
 
+    if (intervals(time).size == 0){
```

Comment:
Rather than doing this here, which makes it impossible to use `Timex` for anything other than intervals, add a method `Timex.allIntervalsFrom` that calls `Timex.allFrom` and filters according to your rules here.